File: tests/fixtures/rust/cases/src/bin/env_vars.rs
use std::env;

fn main() {
    if let Ok(value) = env::var("MOON_FOO") {
        println!("MOON_FOO={}", value);
    }

    if let Ok(value) = env::var("MOON_BAR") {
        println!("MOON_BAR={}", value);
    }

    if let Ok(value) = env::var("MOON_BAZ") {
        println!("MOON_BAZ={}", value);
    }
}

File: tests/fixtures/deno/base/envVars.ts
['MOON_FOO', 'MOON_BAR', 'MOON_BAZ'].forEach((key) => {
	if (Deno.env.get(key)) {
		console.log(`${key}=${Deno.env.get(key).replace(/\\/g, '/')}`);
	}
});
